liquibase-standard/src/main/java/liquibase/parser/core/xml/LiquibaseEntityResolver.java

```diff
@@ -108,7 +108,7 @@ private void warnForMismatchedXsdVersion(String systemId) {
                     String xsdVersion = versionMatcher.group("version");
                     if (!buildVersion.startsWith(xsdVersion)) {
                         hasWarnedAboutMismatchedXsdVersion = true;
-                        String msg = "INFO: An older version of the XSD is specified in one or more changelog's <databaseChangeLog> header. This can lead to unexpected outcomes. If a specific XSD is not required, please replace all XSD version references with \"-latest\". Learn more at https://docs.liquibase.com";
+                        String msg = "INFO: An older version of the XSD is specified in one or more changelog's <databaseChangeLog> header. This can lead to unexpected outcomes. If a specific XSD is not required, please replace all XSD version references with \"-latest\". Learn more at https://docs.liquibase.com/concepts/changelogs/xml-format.html";
```

Comment:
@adrian-velonis1 I think the content of the provided link is ok, I would only suggest making an update of the example provided by showing maybe a small version of one of our xsd files, instead of a changelog.